README.md

```diff
@@ -1,49 +1,28 @@
 # ArcGIS Runtime Toolkit Android
 
-## Components
-Please read the [setup instructions](https://github.com/Esri/arcgis-runtime-toolkit-android/wiki/Setup-Instructions) in our wiki.
+[![doc badge](https://img.shields.io/badge/Doc-purple)](Documentation) [![package badge](https://img.shields.io/bintray/v/esri/arcgis/arcgis-android-toolkit?color=limegreen)](https://bintray.com/esri/arcgis/arcgis-android-toolkit)
 
-- [ArcGISArView](../../wiki/ArcGISArView) - Allows quick and easy integration of AR into your application for a wide variety of scenarios
-- [Compass](../../wiki/Compass) -  Shows the current orientation of a map or scene by displaying a compass icon that points towards North
-- [Scalebar](../../wiki/Scalebar) - Displays the representation of an accurate linear measurement on a map
-- [Bookmarks](Documentation/Bookmarks) - Displays the bookmarks present in a map
+The ArcGIS Runtime Toolkit for Android contains views that you can use with [ArcGIS Runtime SDK for Android](https://developers.arcgis.com/android/).
 
-<!-- 
+You can use the Toolkit in your projects by:
 
-Hiding instructions as AAR is not yet published to Bintray and will not be published to Bintray when 100.6.0 is released.
+1. **Reference from Bintray** - the fastest way to get toolkit into your app
+    * Ensure the Esri public Bintray Maven repository is in your project's gradle file - `https://esri.bintray.com/arcgis`
+    * Add the toolkit dependency in the gradle file - `implementation 'com.esri.arcgisruntime:arcgis-android-toolkit:100.6.1'`
+2. **[Build from source](Documentation/setup)** - do this if you want to customize toolkit
 
-## Instructions
+    ```groovy
+    $ ./gradlew clean assembleDebug --info
+    ```
 
-Update the project Gradle repositories in the **Project** view under **Gradle Scripts > build.gradle (Project:...)** to Esri's Bintray maven repository URL.
+See the [setup instructions](Documentation/setup.md) for a more detailed guide.
 
-```groovy
```

Comment:
are we concluding that the user does not need this snippet or is this covered somewhere else ?

Reply:
I removed the snippet because it is already a pre-requisite for adding the base Runtime component to the app, so theoretically once users get to the toolkit it should already be installed.

Reply:
Makes sense 👍